### DISCUSSION TITLE
I can't seem to get "never combine" working with Windows 11 style taskbar.

### DISCUSSION THREAD

**Discussion by whoistobias** [valinet/ExplorerPatcher#573] [Q&A]
I'd love to use the windows 11 taskbar with icons separated rather than stacked, but the "never combine option only seems to work in the windows 10 style taskbar. Is there a workaround for this or is this just a known limitation?

**Comment by Andre-10** [comment]
Unfortunately, this is only available in Windows 10 like Taskbar
![image](https://user-images.githubusercontent.com/93877359/146429449-e71529f5-6586-4faf-86e5-3c91f23ffe87.png)

**Reply by valinet** [reply]
It's by design, copyright Microsoft. Their stupid decision.

**Comment by Galixte** [comment]
> I'd love to use the windows 11 taskbar with icons separated rather than stacked, but the "never combine option only seems to work in the windows 10 style taskbar. Is there a workaround for this or is this just a known limitation?

@whoistobias A way is to upvote for the proposition n°5, take a look here: https://www.neowin.net/news/here-are-the-top-10-changes-that-windows-11-needs-according-to-the-public/
